src/effects/EffectUI.cpp

```diff
@@ -357,73 +344,46 @@ wxPanel *EffectUIHost::BuildButtonBar(wxWindow *parent, bool graphicalUI)
    {
       S.SetBorder( margin );
 
-      if (!mIsGUI)
-      {
-         mMenuBtn = S.Id( kMenuID )
-            .ToolTip(XO("Manage presets and options"))
-            .AddButton( XXO("&Manage"), wxALIGN_CENTER | wxTOP | wxBOTTOM );
-      }
-      else
+      if (!mwState.expired())
       {
-         mMenuBtn = S.Id( kMenuID )
-            .ToolTip(XO("Manage presets and options"))
-            .Name(XO("&Manage"))
-            .AddBitmapButton( CreateBitmap(effect_menu_xpm, true, true) );
-         mMenuBtn->SetBitmapPressed(CreateBitmap(effect_menu_xpm, false, true));
+         mEnableBtn = S.Id(kEnableID)
+            .Position(wxALIGN_CENTER | wxTOP | wxBOTTOM)
+            .Name(XO("Enable"))
+            .AddBitmapButton(mRealtimeEnabledBM);
+
+         mEnableBtn->SetBitmapDisabled(mRealtimeDisabledBM);
+
+         S.AddSpace( 5, 5 );
       }
 
-      S.AddSpace( 5, 5 );
+      mMenuBtn = S.Id( kMenuID )
+         .ToolTip(XO("Manage presets and options"))
+         .AddButton( XXO("Presets && settings"), wxALIGN_CENTER | wxTOP | wxBOTTOM );
 
       if (!mIsBatch)
       {
-         if (!mIsGUI)
+         if (mSupportsRealtime)
          {
-            if (mSupportsRealtime)
-            {
-               mPlayToggleBtn = S.Id( kPlayID )
-                  .ToolTip(XO("Start and stop playback"))
-                  .AddButton( XXO("Start &Playback"),
-                              wxALIGN_CENTER | wxTOP | wxBOTTOM );
-            }
-            else if (
-               (mEffectUIHost.GetDefinition().GetType() != EffectTypeAnalyze) &&
-               (mEffectUIHost.GetDefinition().GetType() != EffectTypeTool) )
+            if (mwState.expired())
```

Comment:
Wrong condition? After the #3391 mwState should not be used.

Reply:
Done.